html/browsers/the-window-object/named-access-on-the-window-object/window-named-properties.html

```diff
@@ -58,12 +56,6 @@
   assert_false(gsp.hasOwnProperty("constructor"), "gsp.hasOwnProperty(\"constructor\")");
   assert_equals(Object.getOwnPropertyDescriptor(gsp, "constructor"), undefined);
 }, "constructor");
-test(function() {
-  var gsp = Object.getPrototypeOf(Object.getPrototypeOf(window));
-  var names = Object.getOwnPropertyNames(gsp);
-  assert_equals(names.filter((name) => name == "baz").length, 1);
-
-}, "duplicate property names")
```

Comment:
Why change this test? Seems like there was some good coverage?

Reply:
Currently, both Blink and WebKit fail it, and there is no `[[OwnPropertyKeys]]` method specified for [named properties object](https://heycam.github.io/webidl/#named-properties-object). For WebKit, it would be a bit non-trivial to implement, so I am testing its absence in `WebIDL/ecmascript-binding/window-named-properties-object.html`.

Reply:
I see. Still, maintaining the coverage on line 33 seems valuable, by keeping all three frames and testing that... I guess it should be the first one, not the last one, is returned, per https://html.spec.whatwg.org/#document-tree-child-browsing-context-name-property-set ?

Reply:
Right, thanks. I've brought back & updated the test in 4b92f8e.